### PR TITLE
#159951826 Users should be able to like and or dislike articles

### PR DESCRIPTION
**Pull request by AGMETEOR**
#### What does this PR do?
This PR allows users to like and dislike articles they have read. Two buttons are used i.e. thumbs up and thumbs down icons to allow users to like and dislike respectively.
#### What are the relevant pivotal tracker stories?
[#159951826](https://www.pivotaltracker.com/story/show/159951826)
#### Screenshots
```Test Coverage report```
screen shot 2018-09-13 at 23 04 29

### CODE REVIEW

File: src/CSS/Article.css [hunk 1]
```diff
@@ -21,3 +21,30 @@
...
+        left: auto;
+        top: auto;
+        z-index: 77px;
+    }
+    .sticky {
+        position: relative;
+        left: auto;
+        top: auto;
+    }
+}
```
Comment (abulojoshua1) [inline src/CSS/Article.css]: @AGMETEOR, please add EOF on this file.

**Review by abulojoshua1 — CHANGES_REQUESTED** [review]

File: src/containers/Articles/Article.js [hunk 2]
```diff
@@ -2,10 +2,12 @@ import React, { Component } from 'react';
 import axios from 'axios';
 import { NavLink } from 'react-router-dom';
 import PropTypes from 'prop-types';
-import classes from '../../CSS/Article.css';
+// import ReactDOM from 'react-dom';
```
Comment (abulojoshua1) [inline src/containers/Articles/Article.js]: @AGMETEOR, please take out this commented import. I think it will not be required.

**Review by abulojoshua1 — CHANGES_REQUESTED** [review]

File: src/containers/Articles/Article.js (hunk 2 above)
Comment (AGMETEOR) [reply src/containers/Articles/Article.js]: @abulojoshua1 let me take it out.

Comment (abulojoshua1) [reply src/containers/Articles/Article.js]: Ok.

**Review by abulojoshua1 — APPROVED** [review]
Good work @AGMETEOR. The code looks clean and ok!

**Comment by abulojoshua1** [comment]
@AGMETEOR, I have realized that the time stamp on comments in this PR has no spaces, I just seem not to find out what is going on.
screen shot 2018-09-14 at 12 15 44 pm
it should be shown as below instead
screen shot 2018-09-14 at 12 18 11 pm

**Comment by abulojoshua1** [comment]
The message `Lost your way? Go back to login.` has also been modified, The link is now on the next line. (see screenshots below)

The snippet below shows adjustments from this PR
screen shot 2018-09-14 at 12 23 37 pm

The snippet below shows how the message should appear
screen shot 2018-09-14 at 12 21 46 pm

**Comment by AGMETEOR** [comment]
@abulojoshua1 let me take a look. I am not sure why this is happening.

**Review by ssewilliam — APPROVED** [review]

**Pull request merged**